codecov.yml

```diff
@@ -0,0 +1,5 @@
+coverage:
+  status:
+    project:
+      default:
+        threshold: 1
```

Comment:
yeah i had it at first then codecov docs made me change that
https://docs.codecov.io/docs/commit-status#section-threshold
ambiguity wins here